content/about/trademark.fr.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Trademark Policy
```

Comment:
Just a quick typo: title should be `trademark-policy` and not `Trademark Policy`, I know it's minimal, but it's just to guarantee, your file metadata matches the english one!